frontend/src/pages/Admin/Components/Setting/Invite/Invite.jsx

```diff
@@ -3,16 +3,16 @@ import style from "./Invite.module.scss";
 
 export const Invite = () => {
   return (
-    <div className={style.cont}>
+    <div className={style["cont"]}>
       <div className="container">
-        <h1 className={style.h1}>Invite Admins</h1>
-        <div className={style.dash}></div>
+        <h1 className={style["h1"]}>Invite Admins</h1>
```

Comment:
Hey @sakshikhachane I am not able to see this title on the page. Can you please fix it.

Reply:
@Kajol-Kumari I am able to see the title on my end. Can you please share the screenshot?

Reply:
sure!
![Screenshot from 2021-02-25 12-46-07](https://user-images.githubusercontent.com/44888949/109117112-929c8480-7767-11eb-9936-9335538b37ad.png)

Reply:
I have refactored `Invite` and `Manage` subcomponents of `Setting` component of admin. I think there is some confusion.

Reply:
Ah, My bad!! I was checking wrong page :woman_facepalming: